gaphor/ui/icons/Makefile

```diff
@@ -15,6 +15,8 @@ ICONS=diagram \
 	package \
 	interface \
 	association \
+	part-association \
```

Comment:
Should it. E named composite-association instead? It’s more general than parts now.